### ISSUE TITLE
Zooming on iOS

### ISSUE THREAD

**Issue by printfn** [skyjake/lagrange#462]
I’ve been using Lagrange on an iPad, and have had issues with accidental zooming. When I close the app with a five-finger gesture, it often accidentally adjusts the zoom level a bit.

I was wondering if it would maybe be possible to add a setting to disable the two-finger zoom gesture?

**Comment by skyjake** [comment]
You're right, the zoom gesture is not working as it should. I've noticed accidental zooming even while just using my phone normally, bottom-swiping between apps.

I'll adjust the threshold for triggering the zoom, and make sure it gets cancelled if more than two fingers are detected.

**Comment by skyjake** [comment]
Closing this old issue. A while ago, a mandatory delay was added so the pinch is not as easy to trigger.
